File: C#-Basics/07-Exercises/Exercise-02.cs
/* Crea un programa que encuentre:
El número mayor.
El número menor.
La suma de todos los números. */
partial class Program
{
    static void Exercise02()
    {
        List<int> numeros = new List<int> { 12, 5, 30, 8, 45, 17, 25, 50 };
        int suma = 0;
        int numMayor = numeros[0];
        int numMenor = numeros[0];

        foreach (int n in numeros)
        {
            if (n > numMayor)
            {
                numMayor = n;
            }
            if (n < numMenor)
            {
                numMenor = n;
            }

            suma += n;
        }
        WriteLine($"Lista: {String.Join(", ", numeros)}");
        WriteLine($"Número Mayor: {numMayor}");
        WriteLine($"Número Menor: {numMenor}");
        WriteLine($"Suma total: {suma}");
    }
}

File: JS-Basics/09-Exercises/Exercises/Exercise-02.js
const numeros = [34, 12, 56, 7, 89, 23];

let menor = numeros[0];

for (let i = 0; i < numeros.length; i++) {

    if (numeros[i] < menor) {
        menor = numeros[i];
    }

}

console.log("El número menor es: ", menor);
